core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1208,12 +1220,12 @@ public void testRecoveryUncommittedCorruptedCheckpoint() throws IOException {
         TranslogConfig config = translog.getConfig();
         Path ckp = config.getTranslogPath().resolve(Translog.CHECKPOINT_FILE_NAME);
         Checkpoint read = Checkpoint.read(ckp);
-        Checkpoint corrupted = new Checkpoint(0, 0, 0, SequenceNumbersService.UNASSIGNED_SEQ_NO);
+        Checkpoint corrupted = new Checkpoint(0, 0, 0, SequenceNumbersService.NO_OPS_PERFORMED, SequenceNumbersService.NO_OPS_PERFORMED, SequenceNumbersService.UNASSIGNED_SEQ_NO);
```

Comment:
I pushed e6b356d885d22524bc899987525d6e655fffd31a although I left unsettled the issue of the default values for the min and max sequence number (I will address these later).